dvc/progress.py

```diff
@@ -2,8 +2,8 @@
 from __future__ import print_function
 import logging
 import sys
-from tqdm import tqdm
 from concurrent.futures import ThreadPoolExecutor
+from tqdm import tqdm
```

Comment:
why change the import order?

Reply:
`pylinter` was suggesting that. I think PEP8 naming conventions require to follow these import rules

Reply:
@kurianbenoy I don't think that should be in this PR. `dvc` uses `black` and `flake8`. If you think we should use `pylinter` as well then maybe open a new issue so we can consistently apply it across the whole codebase. Right now changing lines based on style just produces unneeded churn.

Reply:
Ok, I agree with you @casperdcl. Will make change to it soon